### ISSUE TITLE
Support policies like "allow every atSign to connect to `someHost:somePort`"

### ISSUE THREAD

**Issue by gkc** [atsign-foundation/noports#1563]
### Is your feature request related to a problem? Please describe.

For demonstration / kicking the tyres purposes, we want to support policies like "allow every atSign to connect to `someHost:somePort`" - we have no way of doing that currently

### Describe the solution you'd like

For demonstration / kicking the tyres purposes, we want to support policies like "allow every atSign to connect to `someHost:somePort`"

**Comment by cpswan** [comment]
How do we stop this from being abused?

**Comment by gkc** [comment]
@XavierChanth can you add a little more background for this feature request please?

**Comment by xavierchanth** [comment]
I don't think this should be part of the open source code or built into the policy engine. The main purpose for it is to serve a public trial device over NoPorts. 

There aren't any clearly defined use cases for having an unrestricted policy anywhere else.

**Comment by gkc** [comment]
Implemented a custom Policy service for the "try it out" part of the getting started journey
